model/src/main/proto/spine_examples/chatspn/message/removal_events.proto

```diff
@@ -69,6 +71,7 @@ message MessageRemovalFailed {
 
 // The message has been marked as deleted.
 message MessageMarkedAsDeleted {
+    option (is).java_type = "io.spine.examples.chatspn.message.MessageRemovalAware";
```

Comment:
Again, it's not obvious what the difference between `MessageRemovalAware` and `MessageRemovalSignal`.

Let's discuss vocally.